components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/internal/APIManagerComponent.java

```diff
@@ -206,7 +206,6 @@ protected void activate(ComponentContext componentContext) throws Exception {
             bundleContext.registerService(Notifier.class.getName(),new GoogleAnalyticsNotifier(),null);
             bundleContext.registerService(Notifier.class.getName(),new ExternalGatewayNotifier(),null);
             bundleContext.registerService(Notifier.class.getName(),new ExternallyDeployedApiNotifier(),null);
-            bundleContext.registerService(Notifier.class.getName(), new CorrelationConfigNotifier(), null);
```

Comment:
Should this line be removed?

Reply:
seems this has been removed while resolving conflicts Reverted the previous commit and Added the changes again for the corresponding commit